src/annotator/plugin/cross-frame.js

```diff
@@ -0,0 +1,112 @@
+// @ts-expect-error - `Plugin` base class is still written in CoffeeScript
+import Plugin from '../plugin';
+
+import AnnotationSync from '../annotation-sync';
+import Bridge from '../../shared/bridge';
+import Discovery from '../../shared/discovery';
+import * as frameUtil from '../util/frame-util';
+import FrameObserver from '../frame-observer';
+
+/**
+ * @typedef {import('../../types/annotator').AnnotationData} AnnotationData
+ */
+
+/**
+ * `CrossFrame` provides a connection from the annotator to the sidebar.
+ *
+ * It can be used to publish events to and subscribe to events from the sidebar.
+ *
+ * This class also has logic for injecting Hypothesis into iframes that
+ * are added to the page if they have the `enable-annotation` attribute set
```

Comment:
That is, the `iframe` element has an `enable-annotation` attribute?

Reply:
Yes.